angular/core/components/group_page/group_theme/group_theme.haml

```diff
@@ -1,6 +1,6 @@
 .group-theme
-  .group-theme__cover{ng-style: "coverStyle()"}
-    .group-theme__upload-photo{ng-mouseover: "themeHoverIn()", ng-mouseleave: "themeHoverOut()", ng-if: "canUploadPhotos()"}
+  .group-theme__cover.group-theme__trigger-upload{ng-style: "coverStyle()", ng-mouseover: "themeHoverIn()", ng-mouseleave: "themeHoverOut()", ng-if: "canUploadPhotos()"}
```

Comment:
Hey @gdpelican, this doesn't work because the `group-theme__header` element also needs to trigger the upload prompt and `group-theme__header` and `group-theme__upload-photo` aren't siblings or anything.